agentassist/src/app/pages/dock-status/dock-status.component.ts

```diff
@@ -138,7 +138,7 @@ export class DockStatusComponent implements OnInit {
   removeRecord(task, index) {
     if (task._id) {
       // this.statusDockerLoading = true;
-      let streamId = this.workflowService.getCurrentBt()._id
+      let streamId = this.authService.smartAssistBots.map(x=>x._id);
```

Comment:
@PrateekSa-Kore did you test these changes in your local??. is the API call is working after these changes

Reply:
@SandeepBa-Kore  yes its working fine in the local...tested properly.